os/generic_unix_base.h

```diff
@@ -25,7 +25,7 @@
 #endif
 #include <sys/param.h>
 
-#if __has_include(<sys/cdefs.h>)
+#if __has_include(<sys/cdefs.h>) && (defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__) || defined(__GLIBC__))
```

Comment:
I don't see why we're making this particular change. Musl (at least the copy I'm looking at) doesn't have `<sys/cdefs.h>`, so wouldn't have tried to include it anyway.